conf/machine/hikey-32.conf

```diff
@@ -0,0 +1,45 @@
+#@TYPE: Machine
+#@NAME: LeMaker HiKey machine
+#@DESCRIPTION: LeMaker HiKey machine
+
+require conf/machine/include/tune-cortexa8.inc
+DEFAULTTUNE = "cortexa8hf-neon"
+
+PREFERRED_PROVIDER_virtual/xserver ?= "xserver-xorg"
+
+XSERVER ?= "xserver-xorg \
+            mesa-driver-swrast \
+            xf86-input-evdev \
+            xf86-input-mouse \
```

Comment:
This is a change adopted from hikey.conf. I suggest to fix this in hikey.conf first and than update this patch.